java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md

```diff
@@ -125,4 +125,69 @@ bin/start.sh \
 --templateProperty kafka.bq.table=kafkaevents \
 --templateProperty kafka.bq.temp.gcs.bucket=templates-demo-kafkatobq \
 --templateProperty kafka.bq.await.termination.timeout=1200000
+```
+
+
+## 2. Kafka To GCS
```

Comment:
You should specify the behaviour of this template as well.
Like how many kafka messages will be written in gcs output file.
What is buffer time window and buffer message limit to flush out these messages into a gcs file.
Will it update existing file or create a new file.
If windowed writes are not being done then we should create a new gitissue to enhance template to add the functionality.

Reply:
Template support default micro-batches.
I am introducing time based micro batching to better control output file generation.
I will be adding the detailed template behaviour post these additions

Reply:
Micro batching  configurations will be introduced in followup git issue.